django_kat/simqueue/migrations/0001_initial.py

```diff
@@ -13,51 +13,124 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Simulation',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
```

Comment:
By definition a migration should not change. So this is just bad ;)

Probably good to just stat over with the migrations, not sure how to solve this properly.